### PR TITLE
GPG-779: Return all postcodes in all caps

### PR DESCRIPTION
**Pull request by xonde**
Test
Went onto the employer details page and manage org page to make sure the postcode was in all caps. When this is deployed to dev we will be testing the downloaded CSVs

Risk
Since the Postcode automatic getter is still public devs in the future could access it that way, however, this is necessary for setting up entity framework

Docs
N/A

### CODE REVIEW

File: GenderPayGap.Database/GpgDatabaseContext.OnModelCreating.cs [hunk 1]
```diff
@@ -49,7 +49,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     entity.Property(e => e.County).HasMaxLength(100);
 
                     entity.Property(e => e.PoBox).HasMaxLength(30);
-
+
```
Comment (malinaslevoaca1) [inline GenderPayGap.Database/GpgDatabaseContext.OnModelCreating.cs:52]: Could you please revert this? 😁

File: GenderPayGap.Database/Models/OrganisationAddress.cs [hunk 2]
```diff
@@ -56,5 +56,10 @@ public OrganisationAddress()
 
         public virtual ICollection<UserOrganisation> UserOrganisations { get; set; }
 
+        public string GetPostCodeInAllCaps()
+        {
+            return PostCode != null ? PostCode.ToUpper() : PostCode;
```
Comment (malinaslevoaca1) [inline GenderPayGap.Database/Models/OrganisationAddress.cs:61]: What about `PostCode?.ToUpper()`?

**Review by malinaslevoaca1 — COMMENTED** [review]
Nice work

File: GenderPayGap.Database/Models/OrganisationAddress.cs (hunk 2 above)
Comment (xonde) [reply GenderPayGap.Database/Models/OrganisationAddress.cs:61]: ah didn't realise C# had that

File: GenderPayGap.Database/GpgDatabaseContext.OnModelCreating.cs (hunk 1 above)
Comment (xonde) [reply GenderPayGap.Database/GpgDatabaseContext.OnModelCreating.cs:52]: ofc haha

**Review by malinaslevoaca1 — APPROVED** [review]

**Review by malinaslevoaca1 — DISMISSED** [review]

**Review by malinaslevoaca1 — APPROVED** [review]

**Pull request merged**